### PR TITLE
Sight Blocker Judge's Responsibility

### PR DESCRIPTION
**Pull request by CarterKoala**
It is the judge's responsibility that competitor does not have a potential line of sight to the puzzle in all cases

### CODE REVIEW

**Comment by OJCubing** [comment]
I am in favour of both this change and the one for competitors  #17

> In this case, the competitor must place the object between their face and the puzzle. Penalty: disqualification of the attempt (DNF).  If this does not completely block the space between the competitor's face and the puzzle, the judge should place an additional sight blocker in that space.

File: wca-regulations.md [hunk 1]
```diff
@@ -482,7 +482,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
     - B4b) The competitor must not apply moves or intentional changes in alignment to the puzzle before they have fully donned the blindfold. Penalty: disqualification of the attempt (DNF).
     - B4c) As soon as the competitor applies the first move to the puzzle, the judge must ensure that there is a sight blocker (e.g. a sheet of paper or cardboard) between the competitor's face and the puzzle.
         - B4c1) In all cases, the competitor must wear the blindfold such that their view of the puzzle would still clearly be blocked if the sight blocker were not in the way.
```
Comment (ohexter) [inline wca-regulations.md:484]:
```suggestion
        - B4c1) In all cases, the competitor must wear the blindfold such that their view of the puzzle would still clearly be blocked if the sight blocker were not in the way.
        - B4c2) A sight blocker must be positioned such that the competitor's view of the puzzle would still clearly be blocked if they were not wearing a blindfold.
```

Comment (ohexter) [reply wca-regulations.md:484]: Is is OK to re-use a previously deleted reg reference in this way?

File: wca-regulations.md [hunk 2]
```diff
@@ -482,7 +482,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
...
-        - B4c3) If the judge and competitor agree beforehand, the competitor may choose to place the puzzle behind a suitable object (e.g. a music stand, the surface of the table) by themselves during the blindfolded phase.
+        - B4c3) If the judge and competitor agree beforehand, the competitor may choose to place the puzzle behind a suitable object (e.g. a music stand, the surface of the table) by themselves during the blindfolded phase. If this does not completely block the space between the competitor's face and the puzzle, the judge should place an additional sight blocker in that space.
```
Comment (ohexter) [inline wca-regulations.md]:
```suggestion
        - B4c3) If the judge and competitor agree beforehand, or if required by the organization team, a suitable object (e.g. a music stand, the surface of the table) placed by the competitor may be used as the sight blocker. If this is not positioned such that the competitor's view of the puzzle would still clearly be blocked if they were not wearing a blindfold, the judge should place an additional sight blocker to achieve that aim.
```

File: wca-regulations.md (hunk 1 above)
Comment (CarterKoala) [reply wca-regulations.md:484]: I think it would be fine, but I don't think this specific addition is necessary

Comment (ohexter) [reply wca-regulations.md:484]: The existing "between the competitor's face and the puzzle" in Bc4 is not particularly precise, and I wanted to establish the sight blocker is trying to establish the same aim as the blindfold (actually obstructing line of sight).

**Pull request merged**